components/Footer.vue

```diff
@@ -1,26 +1,46 @@
 <template>
-  <v-footer class="sFooter" dark padless>
-    <v-container fluid class="bgOrange white--text text-center">
-      <v-row justify="center" class="pa-2" no-gutters>
-        <LogoImeUsp />
-      </v-row>
-      <v-row justify="center" class="pa-2" no-gutters>
-        <v-btn
-          v-for="icon in icons"
-          :key="icon.name"
-          :href="icon.link"
-          class="mx-1 white--text"
-          icon
-          large
-        >
+  <v-footer class="sFooter bgGray" dark padless>
+    <v-container fluid class="mx-3 bgGray white--text text-center">
```

Comment:
Como esse ano vamos fazer um rodapé com alinhamento direito e esquerdo, seria legal que as bordas estivessem alinhadas com o container das páginas. A propriedade `fluid` do `v-container` aumenta ele um pouquinho mais que a das páginas e sem ela as coisas ficam mais certinhas!

O `v-footer`, por padrão, joga o _container_ pra esquerda. Para alinhá-lo ao centro (para que tenhamos as bordas direita e esquerda), transformamos em um flex-box e adicionamos a propriedade justify-center com as classes [Flex Helpers](https://vuetifyjs.com/en/styles/flex) (`d-flex justify-center`).

```suggestion
  <v-footer class="sFooter bgGray d-flex justify-center" dark padless>
    <v-container class="bgGray white--text text-center">
```